packages/paypal-commerce-integration/src/paypal-commerce-fastlane/paypal-commerce-fastlane-payment-strategy.ts

```diff
@@ -340,6 +340,25 @@ export default class PaypalCommerceFastlanePaymentStrategy implements PaymentStr
         };
     }
 
+    /**
+     *
+     * Getting PayPal Fastlane Payment Token
+     *
+     */
+    private async getPayPalFastlanePaymentToken() {
```

Comment:
I don't think we need this method since it used only in one place, so this change could be reverted